docs/man/upslog.txt

```diff
@@ -78,6 +78,11 @@ upslog will run in the background, regardless of logging target.
 Monitor this UPS.  The format for this option is
 +upsname[@hostname[:port]]+.  The default hostname is "localhost".
 
+*-m* 'ups'::
```

Comment:
I guess `-m 'tuple'` sits best here, to keep it short and in sync with updated `usage()`